src/app/recently-added/recently-added.component.ts

```diff
@@ -0,0 +1,71 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
+import { ToastrService } from 'ngx-toastr';
+import { CollectionTag } from '../_models/collection-tag';
+import { Pagination } from '../_models/pagination';
+import { Series } from '../_models/series';
+import { Library } from '../_models/library';
+import { User } from '../_models/user';
+import { ActionFactoryService, ActionItem } from '../_services/action-factory.service';
+import { SeriesService } from '../_services/series.service';
+import { CollectionTagService } from '../_services/collection-tag.service';
+import { AccountService } from '../_services/account.service';
+
+/**
+ * This component is used as a standard layout for any card detail. ie) series, in-progress, collections, etc. 
+ */
+@Component({
+  selector: 'app-recently-added',
+  templateUrl: './recently-added.component.html',
+  styleUrls: ['./recently-added.component.scss']
+})
+export class RecentlyAddedComponent implements OnInit {
+
+  isLoading: boolean = true;
+  collections: CollectionTag[] = [];
+  recentlyAdded: Series[] = [];
+  series: Array<Series> = [];
```

Comment:
A lot of variables from when copied from AllCollections component. Please remove any that aren't needed for this page.